discovery-server/src/main/resources/application.properties

```diff
@@ -1,7 +1,10 @@
 eureka.instance.hostname=localhost
 eureka.client.register-with-eureka=false
 eureka.client.fetch-registry=false
```

Comment:
Move this also to yml so that it will be consistent across all the applications

Reply:
Sure